docs/tooling/cli-guides/create-a-validator.md

```diff
@@ -0,0 +1,87 @@
+# Start a validator with 1-command
```

Comment:
Was the rule of Title Case for section header changed? Why is there no error for this header?

Reply:
@sukantoraymond Please change this line to use Title Case.